nixos/modules/services/misc/omnom.nix

```diff
@@ -227,7 +243,10 @@ in
         "${cfg.dataDir}/templates"."L+" = settings // {
           argument = "${cfg.package}/share/templates";
         };
-        "${cfg.settings.storage.root}"."d" = settings;
+        "${cfg.dataDir}/static"."C" = settings // {
+          argument = "${cfg.package}/share/static";
```

Comment:
What will happen if `share/static` in package is updated? Will running `nixos-rebuild switch` update these static files?

Reply:
You're right, I don't think it will be updated. I encountered this sort of thing [once before](https://github.com/ngi-nix/ngipkgs/blob/f9aa86b77d5eb7e3e56802abf6c32e97c5749510/projects/Openfire/module.nix#L112) and can probably do something similar with this where the `static_dir` will essentially be deleted and re-copied when the version changes. I'm curious if there any better ways, though.

Reply:
We could alternatively discuss this with upstream before we do something like this and make `static_dir` really static, then we can just point it to the nix store without having to complicate things.